### PR TITLE
Fix Todo

### PR DESCRIPTION
**Pull request by sayedihashimi**
The Todo app was producing build warnings. I made the following changes to resolve the warnings.

 - Remove launchscreen.storyboard from the iOS project. It wasn't being used and it had an error that was causing a build warning.
 - Update sqlite package, the version previously installed was producing a build warning.

### CODE REVIEW

File: Todo/Todo.iOS/Todo.iOS.csproj [hunk 1]
```diff
@@ -172,9 +172,6 @@
       <Name>Todo</Name>
     </ProjectReference>
   </ItemGroup>
-  <ItemGroup>
-    <InterfaceDefinition Include="Resources\LaunchScreen.storyboard" />
```
Comment (conceptdev) [inline Todo/Todo.iOS/Todo.iOS.csproj]: a launchscreen storyboard tells the operating system that the app can adjust to multiple screen resolutions - it should not be deleted

with launch screen:
screen shot 2018-03-15 at 2 07 06 pm

without launchscreen:
screen shot 2018-03-15 at 2 07 23 pm

Comment (sayedihashimi) [reply Todo/Todo.iOS/Todo.iOS.csproj]: The LaunchScreen was not configured in `info.plist`. I'll update this to re-add it, add a reuse id to fix the warning, and configure Info.plist.

**Comment by sayedihashimi** [comment]
@conceptdev I have added the launch screen in commit 71f4971

**Comment by davidbritch** [comment]
You've updated sqllite-net-pcl to 1.4.118 in the iOS and Android projects, but not in the PCL and the UWP projects. Could you also update it in those projects please.

In addition, could you also rebase your PR from master in order to avoid the merge conflict in Todo/Todo.Android/packages.config

Thanks!

**Comment by sayedihashimi** [comment]
@davidbritch I rebased master and now there looks to be a bunch of changes in this PR. Is that what you wanted? If not I may have to abandon this PR.

**Pull request merged**